taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/TestCompletedTaskRetrieval.java

```diff
@@ -169,34 +172,35 @@ void completion_of_taskana_task_with_new_process_variables_should_set_these_vari
     // retrieve and check taskanaTaskId
     List<TaskSummary> taskanaTasks =
         this.taskService.createTaskQuery().externalIdIn(camundaTaskIds.get(0)).list();
-    assertEquals(1, taskanaTasks.size());
+    assertThat(taskanaTasks).hasSize(1);
     String taskanaTaskExternalId = taskanaTasks.get(0).getExternalId();
-    assertEquals(taskanaTaskExternalId, camundaTaskIds.get(0));
-
-    Task taskanaTask = this.taskService.getTask(taskanaTasks.get(0).getId());
+    assertThat(taskanaTaskExternalId).isEqualTo(camundaTaskIds.get(0));
 
     // create map for new process variables and set new process variables in it
     Map<String, String> newProcessVariables = new HashMap<>();
 
-    String newProcessVariablesJson =
-        "{\"attribute1\":{\"type\":\"Object\",\"value\":"
-            + "\"{\\\"stringField\\\":\\\"\\\\fForm feed \\\\b Backspace \\\\t Tab"
-            + " \\\\\\\\Backslash "
-            + "\\\\n newLine \\\\r Carriage return \\\\\\\" DoubleQuote\\\",\\\"intField\\\":1,"
-            + "\\\"doubleField\\\":1.1,\\\"booleanField\\\":false,"
+    newProcessVariables.put(
```

Comment:
perfect we do have a test for a complex object. but shouldn't we have a test for a simple String as well?